app/javascript/dashboard/assets/scss/_variables.scss

```diff
@@ -67,6 +67,14 @@ $masked-bg: rgba(0, 0, 0, .4);
 $color-primary-light: #c7e3ff;
 $color-primary-dark: darken($color-woot, 20%);
 
+// Social and inboxes brand colors
```

Comment:
We are trying to reduce scss variable usage to css variables, Can You separate this into a new file `brand-colors`
 like `app/javascript/shared/assets/stylesheets/colors.scss` here ?

Reply:
Why though? These variables are relevant only to the dashboard and not to the rest of the app. Like widgets etc.

Reply:
We are moving styles to Vue components, So don't have to scss files everytime.